tracekit.js

```diff
@@ -149,27 +160,31 @@ TraceKit.report = (function reportModuleWrapper() {
      * @memberof TraceKit.report
      * @throws An exception if an error occurs while calling an handler.
      */
-    function notifyHandlers(stack, isWindowError, error) {
+    function notifyHandlers(stack, isWindowError, aArguments) {
```

Comment:
What does arguments contain? Can we call this args? and maybe a comment to break down what the possible structure is.